ports/nrf/common-hal/rotaryio/IncrementalEncoder.c

```diff
@@ -51,15 +58,17 @@ static void _intr_handler(nrfx_gpiote_pin_t pin, nrf_gpiote_polarity_t action) {
 
     // logic from the atmel-samd port: provides some damping and scales movement
     // down by 4:1.
-    if (self->quarter >= 4) {
-        self->position++;
-        self->quarter = 0;
-    } else if (self->quarter <= -4) {
-        self->position--;
-        self->quarter = 0;
+    if (new_state == 2 && self->quarter != 0) {
```

Comment:
if a different encoder (with detent logic levels of pin_a=0 and pin_b=0) needs to be supported, this line of code would effectively need to check `new_state == detent_decimal_gray_code` where detent_decimal_gray_code = 0. not a hard coded '2'

for now this PR is not accommodating configuration of a detent_decimal_gray_code 😊